docs/README-maintainers.md

```diff
@@ -14,94 +14,73 @@ Guidelines for the core maintainers of the ciscopuppet project - above and beyon
-    ```
-    
-4. Commit your changes and push the release branch to GitHub for review by Cisco and PuppetLabs:
-
-	```
-	git flow release publish 1.0.1
-	```
-	
-5. Once Cisco and PuppetLabs are in agreement that the release branch is sane, finish the release and push the finished release to GitHub:
-
-    ```
-    git flow release finish 1.0.1
-    git push origin master
-    git push origin develop
-    git push --tags
-    ```
-
-6. Add release notes on GitHub, for example `https://github.com/cisco/cisco-network-puppet-module/releases/new?tag=v1.0.1`. Usually this will just be a copy-and-paste of the relevant section of the `CHANGELOG.md`.
-
-7. Reach out to PuppetLabs to publish the new module version to PuppetForge.
+1. Pull release branch based on the `develop` branch.
+      * 0.0.x - a bugfix release
+      * 0.x.0 - new feature(s)
+      * x.0.0 - backward-incompatible change (if unvoidable!)
+
+1. Run full beaker test regression on [supported platforms.](https://github.com/cisco/cisco-network-puppet-module#resource-platform-support-matrix)
+     * Fix All Bugs.
+     * Make sure proper test case skips are in place for unsupported platforms.
+     * Ensure that tests have been executed against released Gem versions (release a new version if necessary!) and do not have dependencies on unreleased Gem code.
+
+
+1. Update [changelog.](https://github.com/cisco/cisco-network-puppet-module/blob/develop/CHANGELOG.md)
+     * Make sure CHANGELOG.md accurately reflects all changes since the last release.
+     * Add any significant changes that weren't documented in the changelog
+     * Clean up any entries that are overly verbose, unclear, or otherwise could be improved.
+     * Indicate new platform support (if any) for exisiting providers.
+     * Create markdown release tag.
+       ```diff
+       -## [Unreleased]
+       +## [1.0.1] - 2015-08-28
+       ...
+       +[1.0.1]: https://github.com/cisco/cisco-network-puppet-module/compare/v1.0.0...v1.0.1
+       [1.0.0]: https://github.com/cisco/cisco-network-puppet-module/compare/v0.9.0...v1.0.0
+       ```
+
+1. Update [metadata.json](https://github.com/cisco/cisco-network-puppet-module/blob/develop/metadata.json) file.
+     * Update Version
+       ```diff
+          "name": "puppetlabs-ciscopuppet",
+       -  "version": "1.0.0",
+       +  "version": "1.0.1",
+          "author": "cisco",
+       ```
+     * Update Supported OS Verions (if applicable)
+
+1. Update [`cisco_node_utils.rb` rec_version = Gem::Version.new('x.x.x')](https://github.com/cisco/cisco-network-puppet-module/blob/develop/lib/puppet/feature/cisco_node_utils.rb#L40) version.
+
+1. Verify/Update [netdev_stdlib version](https://github.com/cisco/cisco-network-puppet-module/blob/develop/metadata.json#L11) requirement if needed.
+
+1. Verify puppet module can be built using the [new puppet module version](https://github.com/cisco/cisco-network-puppet-module/blob/develop/metadata.json#L3).
+
+1. Scrub README Docs.
```

Comment:
Put a hyperlink to README.md (https://github.com/cisco/cisco-network-puppet-module/blob/develop/README.md)